test/SmokeTest.Tests/SmokeTest.Tests.csproj

```diff
@@ -31,10 +31,7 @@
 
   <!-- Ensure our data files are copied next to the test executables. -->
   <ItemGroup>
-    <None Update="testdata\data.json" CopyToOutputDirectory="PreserveNewest" />
-    <None Update="testdata\policy.rego" CopyToOutputDirectory="PreserveNewest" />
-    <None Update="testdata\simple\system.rego" CopyToOutputDirectory="PreserveNewest" />
-    <None Update="testdata\weird_name.rego" CopyToOutputDirectory="PreserveNewest" />
+    <None Update="testdata\**\*" CopyToOutputDirectory="PreserveNewest" />

```

Comment:
Yay

Reply:
My thoughts exactly! :slightly_smiling_face: Adding each individual file was quite a pain. At least now that's limited to just the test fixture code, and doesn't also involve the build system anymore.